charts/catalog/templates/apiserver-deployment.yaml

```diff
@@ -20,6 +20,7 @@ spec:
         release: "{{ .Release.Name }}"
         heritage: "{{ .Release.Service }}"
     spec:
+      serviceAccountName: server
```

Comment:
sheesh. You mentioned it and tagged it for me and I *still* missed it. sorry guys.